frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -165,9 +176,39 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({
     setNodes(preSelectedNodes);
   }
 
+  const validateCPU = (node: formattedNodeType) => {
```

Comment:
I've change the name to make it clear it's returning a boolean.

```suggestion
  const hasMinimumCPU = (node: formattedNodeType): boolean => {
```